### PR TITLE
refactor(renderer): Extract the renderer to a new library

### PR DESCRIPTION
**Pull request by danpersa**
affects: tessellate-fragment, tessellate-renderer

### CODE REVIEW

**Comment by mfellner** [comment]
Looks good so far. I'd like to add a commit to clean up the dependencies a bit and to fix the failing tests.

**Comment by danpersa** [comment]
Please go ahead :)

File: packages/tessellate-fragment/.gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
 /dist
+test/__snapshots__
```
Comment (mfellner) [inline packages/tessellate-fragment/.gitignore]: Fixed in aa9a580

Comment (danpersa) [inline packages/tessellate-fragment/.gitignore]: ok

**Comment by danpersa** [comment]
👍

**Pull request merged**